features/quizmaster/view_quiz_page.feature

```diff
@@ -5,7 +5,7 @@ Feature: As a Quizmaster
 
 Background:
   Given there is a quiz called "Trivia"
-  And there is a team named "Craft Academy"
+  And there is a team named "Craft Academy" playing "Trivia"
```

Comment:
I did it like this to make sure the association between team and quiz were happening. Before this feature our Factory was assigning a random quiz for the team to have. It wasn't a problem before but was now (I think... it's been a while).